src/main/java/com/znsio/teswiz/runner/Driver.java

```diff
@@ -323,9 +323,13 @@ public void selectNotificationFromNotificationDrawer(By selectNotificationLocato
     }
 
     public void putAppInBackground(int duration) {
-        throw new NotImplementedException("To be migrated to appium 2.0");
-        // todo - implement for appium2.0
-//        ((AppiumDriver) driver).runAppInBackground(Duration.ofSeconds(duration));
+        String platform = String.valueOf(Runner.getPlatform());
```

Comment:
Why converting to string. You can compare the enum value directly

Reply:
done